src/components/products/Products.jsx

```diff
@@ -1,5 +1,80 @@
 import React from "react";
+import ProductCard from "../layout/productCard/ProductCard";
+import { Container, Row } from "react-bootstrap";
 
 export default function Products() {
-  return <div></div>;
+  const data = [
```

Comment:
is this temporary until real products are placed? if so, add a comment indicating this